{{cookiecutter.project_name}}/monitoring/SetUpDataMonitoringForInferenceTable.py

```diff
@@ -0,0 +1,131 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC
+# MAGIC # Data Monitoring for Inference Table
+# MAGIC see monitoring/README.md for details
+
+# COMMAND ----------
+
+# Install the Data Monitoring client library
+# TODO: Fill in wheel URL
+%pip install "{wheel_URL}"
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Required Fields
+# MAGIC TODO: Fill in and review required fields
+
+# COMMAND ----------
+
+# The name of the column that represents the inference timestamp (i.e. time of prediction) for this row. This column should be of TimestampType.
+timestamp_col = "ts"
```

Comment:
Since these are all required and we expect the user must fill them in, would it be better to leave them as None and then assert below that all required parameters are not none? This will avoid situations where the user doesnt study this notebook carefully and just runs it (since all required parameters are filled in) and then hits weird errors because it's trying to create a monitor with these irrelevant parameters

Reply:
If user uses batch inference from MLOps stacks, our BatchInference notebook adds the timestamp column with name inference_timestamp.(I'll need to update the column name here)
I can make it optional and add comments about the preset name.

Reply:
Sorry not just for the timestamp column but all required columns. The idea is that we force the user to specify them instead of providing fake values here